.circleci/test_runner.py

```diff
@@ -82,11 +96,61 @@ def get_envs() -> dict:
 def start_workflow():
     ''' Get the party started
     '''
-    logging.info('Workflow started')
-    envs = get_envs()
-    job_id = str(create_job(envs['TEST_RUNNER_SERVICE']))
-    check_job_completed(envs['TEST_RUNNER_SERVICE'], job_id)
-    check_job_result(envs['TEST_RUNNER_SERVICE'], job_id)
+    try:
+        logging.info('Workflow started')
+        envs = get_envs()
+        job_id = str(create_job(envs['TEST_RUNNER_SERVICE']))
+        check_job_completed(envs['TEST_RUNNER_SERVICE'], job_id)
+        check_job_result(envs['TEST_RUNNER_SERVICE'], job_id)
+
+        # Post success to slack
+        success_text = """
+        Functional tests for elixir-omg completed successfully!\n{}
+        """.format(
+            envs['CIRCLE_BUILD_URL']
+        )
+        for webhook in [
+            envs['TEST_RUNNER_SLACK_WEBHOOK_01'],
+            envs['TEST_RUNNER_SLACK_WEBHOOK_02']
+        ]:
+            if webhook is not None:
+                requests.post(webhook, json={
+                    'text': success_text
+                })
+
+    except Exception as e:
```

Comment:
Bare `Exception` should not be used, and the `try` statement should identify and catch one condition only.

Reply:
I've chosen to remove the `try/catch` exception handling in the `create_job`, `check_job_completed`, `check_job_result`, ... and place them into the main function `start_workflow` and have that function act as the main function handler for all of them.

The rational is because the functions `check_job_completed`, `check_job_result`, etc are dealing with IO, and throws a bunch of IO related exceptions such as `TimeOutException`, `ConnectionException`, as well as the exception of the possibility of the job failing. Having a try/catch for each IO scenario in each function is overly verbose for a simple notifications script, imo and is overly complicated when our main goal is to surface information / notify the #dev-plasma team when the tests fail.